deployment/templates/route.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  labels:
+    app: {{ .Values.name }}
+  name: {{ .Values.name }}
+spec:
+  {{- if .Values.hostname }}
+  host: {{ .Values.hostname }}
+  {{- end }}
+  port:
+    targetPort: "{{ .Values.servicePort }}-tcp"
+  tls:
+    termination: edge
+    insecureEdgeTerminationPolicy: Redirect
+  to:
+    kind: Service
+    name: {{ .Values.name }}
+    weight: 100
+  wildcardPolicy: None
+status:
+  ingress: []
```

Comment:
I missed this on activity - i think we need the route off by default in all envs.

Reply:
removed